DiceKeys/Views/Scanning/CameraFrameCountView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CameraFrameCountView.swift
+//  DiceKeys
+//
+//  Created by Kevin Shah on 26/01/21.
+//
+
+import SwiftUI
+
+// This model tracks the frame count in an object that ScanDiceKey needn't
+// observe, so that only the child that needs to render the frame count
+// (CameraFrameCountView, below) refreshes on every frame.
+class CameraFrameCountModel: ObservableObject {
```

Comment:
Please move the models out of the view file. Create a Model folder at the View Folder level and move them there.
Ideally I would suggest DiceKeys > Models > Scanning > CameraFrameCountModel.swift